### PR TITLE
Targeting, Pending targets, and Observed targets

### PR DESCRIPTION
**Pull request by profxj**
## Description of the Change

This PR enables:

- Generating target lists
- Enables ingesting (from `chime_ffff_pz`) targets pending observation
- Enables ingesting an observing log (from `chime_ffff_pz`)

Basically the whole enchilada, more or less.  Not that there isn't a ton
more to do..

## Checklist

Please check all that apply to your proposed changes

- [X ] HTML code change
- [ ] Added package dependency
- [X] Added data
- [X] Changed django model
- [ ] Documentation change

### CODE REVIEW

**Review by davecoulter — APPROVED** [review]
Looks good, X.

File: YSE_App/data_utils.py [hunk 1]
```diff
@@ -1556,6 +1557,7 @@ def ingest_path(request):
       - obs_group (str): name of the instrument; must be present in the
         ObservationGroup table
       - P_Ux (float): Unseen posterior;  added to the transient
```
Comment (tarraneh) [inline YSE_App/data_utils.py:1559]: frb_followup_resource

File: YSE_App/frb_init.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    # Create the FRB ones
+    all_status = [\
+        'Unassigned', # Likely just ingested although most will be given PublicPATH
+        'PublicPATH', # Needs to be run through PATH with public data
+        'Image', # Needs deeper imaging
+        'Spectrum', # Needs spectroscopy for redshift
+        'DeepPATH', # Needs PATH run on deeper imaging
+        'PendingImage', # Pending deeper imaging with an FRBFollowUp
+        'PendingSpectrum', # Pending spectroscopy with an FRBFollowUp
```
Comment (tarraneh) [inline YSE_App/frb_init.py:19]: `PendingMask`?

File: YSE_App/frb_init.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    # Delete all existing
+    TransientStatus.objects.all().delete()
+
+    # Create the FRB ones
+    all_status = [\
+        'Unassigned', # Likely just ingested although most will be given PublicPATH
+        'PublicPATH', # Needs to be run through PATH with public data
+        'Image', # Needs deeper imaging
+        'Spectrum', # Needs spectroscopy for redshift
```
Comment (tarraneh) [inline YSE_App/frb_init.py:16]: I guess there's needing spectroscopy for redshift and needing a mask for foreground analyses. Do we want to differentiate?

**Review by tarraneh — COMMENTED** [review]
Looks good

**Review by tarraneh — APPROVED** [review]

File: YSE_App/frb_init.py (hunk 2 above)
Comment (profxj) [reply YSE_App/frb_init.py:19]: I don't think we need to distinguish here.

File: YSE_App/frb_init.py (hunk 3 above)
Comment (profxj) [reply YSE_App/frb_init.py:16]: FFFF-PZ will not monitor fore-ground follow-up

**Pull request merged**